js/ui/windowManager.js

```diff
@@ -2006,6 +2102,11 @@ var WindowManager = new Lang.Class({
             return;
         }
 
+        // Endless libanimation extension
+        if (this._animationsServer) {
+            actor._animatableSurface = this._animationsServer.register_surface(new ShellWindowManagerAnimatableSurface(actor));
+        }
```

Comment:
Nitpick: I think the style usually is to not use brackets on one line ifs..

Reply:
Fixed

Reply:
It's not really fixed, right?